[report]
fail_under=84
source=vsutil.py
